### PR TITLE
Improve guidance on removing default mappings.

### PR DESCRIPTION
**Pull request by jtibshirani**
In 7.x, an index template will fail to apply if it contains a `_default_`
mapping. Several users have expressed confusion over the fact that loading the
template doesn't show any default mappings. This docs change clarifies that in
order to see all mappings in the template, you must pass `include_type_name`.

Addresses #48427.

### CODE REVIEW

File: docs/reference/migration/migrate_7_0/mappings.asciidoc [hunk 1]
```diff
@@ -31,6 +31,20 @@ of `_id`.
 The `_default_` mapping has been deprecated in 6.0 and is now no longer allowed
 in 7.0. Trying to configure a `_default_` mapping on 7.x indices will result in
 an error.
+
+If an index template contains a `_default_` mapping, it will fail to create new
+indices. To resolve this issue, the `_default_` mapping should be removed from
+the template. Note that in 7.x, the get templates API does not show the
```
Comment (talevy) [inline docs/reference/migration/migrate_7_0/mappings.asciidoc:37]: should the `get templates API` be capitalized `Get Templates API` and maybe hyperlink to the specific docs?

**Review by talevy — APPROVED** [review]
just left a nit that you can feel free to ignore.

LGTM so long as CI is happy!

File: docs/reference/migration/migrate_7_0/mappings.asciidoc (hunk 1 above)
Comment (jtibshirani) [reply docs/reference/migration/migrate_7_0/mappings.asciidoc:37]: 👍

**Comment by jtibshirani** [comment]
@elasticmachine run elasticsearch-ci/1

**Comment by jtibshirani** [comment]
Thanks for the review!

**Pull request merged**